hw/ip/aes/model/aes_model.core

```diff
@@ -0,0 +1,43 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:model:aes:0.5"
+description: "AES Model"
+filesets:
+  files_cc:
+    files:
+      - crypto.c
+      - crypto.h: { is_include_file: true }
+      - aes.c
+      - aes.h: { is_include_file: true }
+    file_type: cppSource
+
+targets:
+  default:
+    default_tool: verilator
```

Comment:
can you compile this module standalone? If not, you can remove all verilator-specific things, and only keep the default target with the filesets.

Reply:
That's a very good point. I can compile the module standalone (but using a separate Makefile). I removed everything not strictly required for the model. The only thing remaining in `verilator_options` are the linker flags required for the crypto library. These are also required for other tools. Is there way to define this e.g. for VCS?